packages/preview/dnd/0.0.2/README.md

```diff
@@ -0,0 +1,238 @@
+# Typst Dungeons and Dragons Template
+
+Create fancy-looking homebrewn Dungeons and Dragons 2024 edition content using
+this [typst](https://www.github.com/typst/typst) template.
+
+Check out the example [document](examples/dnd.typ) and the resulting [PDF file](examples/dnd.pdf).
+
+While there have been many amazing efforts in the past to replicate the official
+Dungeons and Dragons book style, this project has a slightly different aim, best
+summarised by the following goals:
+
+- Easy to use: the template is designed to be easy to use; all you need to know
+  is the typst syntax. The code will not get in the way of your writing.
+- Batteries included, but works out of the box: I hope to make the template as
+  extensible as possible, but without exposing any of that complexity onto users
+  until the very moment they need it.
+- Promote open source projects, where possible: for example, instead of
+  replicating the official fonts, this project uses
+  [Open Font License](https://opensource.org/licenses/OFL-1.1) fonts that match
+  the vibe of the official ones. [^fonts]
+- Optimised for home printing: the official books are printed on high-quality
+  paper, with a lot of ink and using fancy backgrounds. This project aims to
+  look good when printed on a home printer, balancing style with ink usage.
+- Customisable: there is a theme parameter that allows you to change the look of
+  the document, by overriding different aspects like colours, fonts, sizes, etc.
+  In theory, this should allow anyone to create and share themes that match the
+  style of their homebrew world.
+
+[^fonts]: Read more about the font choices in this [document](./docs/fonts.md)
+
+## Example
+
+If you are familiar with Markdown or the MediaWiki syntax, typst won't feel too
+different. You can check out the [typst syntax](https://typst.app/docs/tutorial/)
+for reference. Here is what a very simple document might look like:
+
+```typst
+#import "@preview/typst-dnd-template:0.0.1": dnd
+#show: dnd  // These two lines simply apply the template
+
+= Shadowlands  // Campaign title
+
+== The Shadowy inn  // Heading
+
+// Paragraph
+Deep within the mist-shrouded valleys of the Northern Reach lies the forgotten township of Shadowfen. Once a thriving trading post, it now stands as a haunting reminder of prosperity lost to an ancient curse. As adventurers drawn to this forlorn place, you will uncover secrets long buried and face horrors that lurk in the perpetual twilight.
+```
+
+This would render like this:
+
+![Example render of the sample code](docs/assets/render.png)
+
+## Usage
+
+To use this template, you need to have [typst](https://www.github.com/typst/typst)
+installed. You can check out the [recommended setup](#recommended-setup) for
+the set of tools used when developing this template.
+
+To use the template, simply import it and use it in your typst document:
+
+```typst
+#import "@preview/typst-dnd-template:0.0.1": dnd
```

Comment:
The import here is wrong, please update it once you will have decided on a new package name.